### PR TITLE
Honour guards + Prophets

### PR DESCRIPTION
**Pull request by Stingray540**
Swaps out elite ultra for 2x honour guards and a prophet. The honour guards are heavily restricted in what they can do in order to reduce slaughter.

Adds a staff for the honour guards that does pain damage, mainly used to close distance and scare off enemies (and wack unngoy). Currently using temporary sprites.

### CODE REVIEW

File: code/modules/halo/weapons/covenant/melee.dm [hunk 1]
```diff
@@ -190,6 +191,78 @@
...
+//I had to define lunge again due to the fact I'm not basing this on the /elite_sword/ path.
+
+/obj/item/weapon/honour_staff/proc/get_species_leap_dist(var/mob/living/carbon/human/mob)
+	if(isnull(mob) || !istype(mob))
+		return 0
+	if(mob.species.type in ESWORD_LEAP_FAR_SPECIES)
+		return 7
+	return STAFF_LEAP_DIST
+
+/obj/item/weapon/honour_staff/afterattack(var/atom/target,var/mob/user)
```
Comment (X0-11) [inline code/modules/halo/weapons/covenant/melee.dm]: For the sake of reducing duplication and keeping code consistent, this should not be redefined. If features are present in the /elite_sword subtype that are not needed in this item, find a way to override them.

**Review by X0-11 — CHANGES_REQUESTED** [review]
address comments

File: code/modules/halo/weapons/covenant/melee.dm (hunk 1 above)
Comment (Stingray540) [reply code/modules/halo/weapons/covenant/melee.dm]: This issue has been fixed in the last commit.

Comment (X0-11) [reply code/modules/halo/weapons/covenant/melee.dm]: Item removal isn't fixing it.

Comment (Stingray540) [reply code/modules/halo/weapons/covenant/melee.dm]: I have now fixed it.

**Review by X0-11 — APPROVED** [review]

**Pull request merged**